util/src/org/commcare/util/mocks/CommCareInstanceInitializer.java

```diff
@@ -13,7 +15,7 @@
 /**
  * @author ctsims
  */
-public class CommCareInstanceInitializer extends InstanceInitializationFactory {
+public class CommCareInstanceInitializer implements InstanceInitializationFactory {
```

Comment:
Hm, do we know where the J2ME one of these lives? It should also get changed, right?